pkg/helpers/apply/apply.go

```diff
@@ -159,8 +161,18 @@ func getTemplate(templateName string) *template.Template {
 	return tmpl
 }
 
-func MustTempalteAsset(name, templateName string, reader ScenarioReader, values interface{}) ([]byte, error) {
-	tmpl := getTemplate(templateName)
+//MustTempalteAsset generates textual output for an file name.
+//If a headerFile is specified it will be added as a header of the file.
```

Comment:
I am not quite sure what is the purpose of headerfile. Would you add more comments here?

Reply:
The header file is an extra file you would like to add to the the files... this file can contain template functions... this like having the helm _helpers.tpl.